protocol/messenger_mailserver.go

```diff
@@ -589,10 +588,6 @@ func (m *Messenger) calculateGapForChat(chat *Chat, from uint32) (*common.Messag
 }
 
 func (m *Messenger) canSyncWithStoreNodes() (bool, error) {
-	if m.connectionState.IsExpensive() {
```

Comment:
We don't check if `IsExpensive` anymore for store node syncing?

Reply:
`connectionState` was propagated by an unused endpoint (legacy mobile). In the case of status-app, this is not invoked at all. Moreover, the online logic was ambiguous, as in some cases it was derived from `connectionState`, while in others it came from `messenger.Online()`, which did not use `connectionState` at all :dizzy_face: